site/content/docs/commands/env-init.ja.md

```diff
@@ -29,27 +29,44 @@ Import Existing Resources Flags
       --import-vpc-id string             Optional. Use an existing VPC ID.
 
 Configure Default Resources Flags
-      --override-private-cidrs strings   Optional. CIDR to use for private subnets (default 10.0.2.0/24,10.0.3.0/24).
-      --override-public-cidrs strings    Optional. CIDR to use for public subnets (default 10.0.0.0/24,10.0.1.0/24).
-      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC (default 10.0.0.0/16).
+      --override-az-names strings        Optional. Availability Zone names.
+                                         (default 2 random AZs)
+      --override-private-cidrs strings   Optional. CIDR to use for private subnets.
+                                         (default 10.0.2.0/24,10.0.3.0/24)
+      --override-public-cidrs strings    Optional. CIDR to use for public subnets.
+                                         (default 10.0.0.0/24,10.0.1.0/24)
+      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC.
+                                         (default 10.0.0.0/16)
 
-Global Flags
-  -a, --app string   Name of the application.
+Telemetry Flags
+      --container-insights   Optional. Enable CloudWatch Container Insights.
 ```
 
 ## 実行例
-AWS プロファイルの "default" に、デフォルト設定を使用して test Environment を作成します。
-
+AWS プロファイルの "default" 利用し、デフォルト設定を使用して test Environment を作成します。
 ```bash
 $ copilot env init --name test --profile default --default-config
 ```
 
-AWS プロファイルの "prod-admin" を利用して既存の VPC に prod-iad Environment を作成します。
+AWS プロファイルの "prod-admin" を利用して既存の VPC に prod-iad Environment を作成し、 CloudWatch Container Insights　を有効化します。
```

Comment:
Does this command import an existing VPC? I guess the `env init` command would ask you either of creating a VPC with default configuration or creating a VPC with specified configuration or importing a VPC.

Reply:
The translation of importing an existing VPC is wrong.